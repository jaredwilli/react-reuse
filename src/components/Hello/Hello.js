import React from 'react';
import PropTypes from 'prop-types';

/** A basic example component description for Hello component */
export default function Hello() {
    return (
        <div>
            Hello world
        </div>
    );
};

Hello.propTypes = {
    /** Message to display */
    message: PropTypes.string
};

Hello.defaultProps = {
    message: 'World'
};
