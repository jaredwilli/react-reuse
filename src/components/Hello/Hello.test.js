import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import Hello from './Hello';

describe('Hello', () => {
    it('renders the component correctly', () => {
        const tree = renderer.create(<Hello />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
