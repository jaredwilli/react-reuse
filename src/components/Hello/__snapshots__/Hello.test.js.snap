// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello renders the component correctly 1`] = `
<div>
  Hello world
</div>
`;
